policies/sidebar.shtml

```diff
@@ -22,10 +22,10 @@
 	<a href="secpolicy.html">Security Policy</a>
       </li>
       <li>
-	<a href="omc-bylaws.html">OMC (project) Bylaws</a>
+	<a href="omc-bylaws.html">OpenSSL Bylaws</a>
```

Comment:
Maybe rename the file to `openssl-bylaws.html` ?

Reply:
While I have no objection in principle to this, it turns out to be non-trivial since this page is referenced by quite a few other pages - so I'd rather keep it as-is for now.

Reply:
You can always use the new name, and have your webmaster set up a redirect in .httaccess for the old name.